sdk/samples/core/mosquitto_multiple_rpc_sample.c

```diff
@@ -0,0 +1,629 @@
+
+  az_result rc;
+
+  // infinite execution loop
+  for (int i = 20; !sample_finished && i > 0; i++)
+  {
+    // check for commands for the server
+    LOG_AND_EXIT_IF_FAILED(check_for_commands_to_execute());
+
+    // remove any expired commands from the client
+    pending_command* expired_command = get_first_expired_command(client_pending_commands);
+    while (expired_command != NULL)
+    {
+      printf(LOG_APP_ERROR "command ");
+      print_correlation_id(expired_command->correlation_id);
+      printf(" expired\n");
+      az_result ret = remove_command(&client_pending_commands, expired_command->correlation_id);
+      if (ret != AZ_OK)
+      {
+        printf(LOG_APP_ERROR "Expired command not a pending command\n");
+      }
+      expired_command = get_first_expired_command(client_pending_commands);
+    }
+#ifdef _WIN32
+    Sleep((DWORD)1000);
+#else
+    sleep(1);
+#endif
+    printf(LOG_APP "Waiting...\r");
+    fflush(stdout);
+    // invokes a start module command every 15 seconds and a stop module command every 10 seconds.
+    // This cadence/how it is triggered should be customized for your solution.
+    if (i % 15 == 0)
+    {
+      uuid_t new_uuid;
+      uuid_generate(new_uuid);
+      az_mqtt5_rpc_client_invoke_req_event_data command_data
+          = { .correlation_id
+              = az_span_create((uint8_t*)new_uuid, AZ_MQTT5_RPC_CORRELATION_ID_LENGTH),
+              .content_type = content_type,
+              .rpc_server_client_id = server_client_id,
+              .command_name = start_module_command_name,
+              // TODO: Payload should be generated and serialized
+              .request_payload = AZ_SPAN_FROM_STR(
+                  "{\"RequestTimestamp\":1691530585198,\"RequestedFrom\":\"mobile-app\"}") };
+      LOG_AND_EXIT_IF_FAILED(add_command(
+          &client_pending_commands, command_data.correlation_id, start_module_command_name, 10000));
+      rc = az_mqtt5_rpc_client_invoke_begin(&rpc_client_policy, &command_data);
+      if (az_result_failed(rc))
+      {
+        printf(
+            LOG_APP_ERROR "Failed to invoke command '%s' with rc: %s\n",
+            az_span_ptr(start_module_command_name),
+            az_result_to_string(rc));
+        remove_command(&client_pending_commands, command_data.correlation_id);
+      }
+    }
+    else if (i % 10 == 0)
+    {
+      uuid_t new_uuid;
+      uuid_generate(new_uuid);
+      az_mqtt5_rpc_client_invoke_req_event_data command_data
+          = { .correlation_id
+              = az_span_create((uint8_t*)new_uuid, AZ_MQTT5_RPC_CORRELATION_ID_LENGTH),
+              .content_type = content_type,
+              .rpc_server_client_id = server_client_id,
+              .command_name = stop_module_command_name,
+              // TODO: Payload should be generated and serialized
+              .request_payload = AZ_SPAN_FROM_STR(
+                  "{\"RequestTimestamp\":1691530585198,\"RequestedFrom\":\"mobile-app\"}") };
```

Comment:
add get time for time stamp

Reply:
Will add this when we do proper payload generation/serialization